.editorconfig

```diff
@@ -141,6 +141,26 @@ dotnet_diagnostic.IDE0230.severity = suggestion  # Use raw string literal when b
 # remove this when no-longer supporting pre-9.0 frameworks
 dotnet_diagnostic.IDE0330.severity = none        # Prefer 'System.Threading.Lock'
 
+# Roslynator analyzer suppressions (per tests/README.md)
```

Comment:
Do a cleanup of this referenced `tests/README.md` file accordingly, since much of the information about Roslynator was mostly ephemeral notes; leave only what is needed for long-lived developer information.